controller/auth_utility_test.go

```diff
@@ -23,9 +23,9 @@ import (
 const (
 	// the various HTTP endpoints
 	endpointWorkItems              = "/api/workitems"
-	endpointWorkItemTypes          = "/api/workitemtypes"
+	endpointWorkItemTypes          = "/api/spaces/%s/workitemtypes"
 	endpointWorkItemLinkCategories = "/api/workitemlinkcategories"
-	endpointWorkItemLinkTypes      = "/api/workitemlinktypes"
+	endpointWorkItemLinkTypes      = "/api/spaces/%s/workitemlinktypes"
```

Comment:
Should we evaluate a patch to goa to generate up app.Hrefxxxx for all GET actions, not only 'show'